scripts/jenkins/qa_devstack.sh

```diff
@@ -107,14 +113,28 @@ function h_setup_extra_disk {
 }
 
 function h_setup_devstack {
-    $zypper in git-core which ca-certificates-mozilla net-tools git-review
-    $zypper in 'group(nogroup)'
+    $zypper in git-review
+    $zypper mr -d SUSE_Factory_Head
+    $zypper in git-core which ca-certificates-mozilla net-tools
```

Comment:
This should really be part of the image. Which broken image is that? JeOS?

Reply:
[**@dirkmueller**](https://github.com/dirkmueller) commented on [null](https://github.com/SUSE-Cloud/automation/pull/3108/files#r250491565 "2019-01-24T07:37:49Z - Replied by Github Reply Comments"):
> This should really be part of the image.

Which package are you referring to?  But I didn't touch this line, it was already there, and only ensures that these packages are present.

> Which broken image is that? JeOS?

Nope, there was no brokenness here.  This was a straightforward SP3 bare metal install onto one of the SEV nodes.  It only installed git-core and updated ca-certificates-mozilla and net-tools which were already there.